cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectBuild.java

```diff
@@ -81,6 +83,19 @@ public static KafkaConnectBuild fromCrd(KafkaConnect kafkaConnect, KafkaVersion.
 
         if (spec.getBuild() != null)    {
             validateBuildConfiguration(spec.getBuild());
+
+            // The additionalKanikoOptions are validated separately to avoid parsing the list twice
+            if (spec.getBuild().getOutput() != null
+                    && spec.getBuild().getOutput() instanceof DockerOutput) {
+                DockerOutput dockerOutput = (DockerOutput) spec.getBuild().getOutput();
+
+                if (dockerOutput.getAdditionalKanikoOptions() != null
+                        && !dockerOutput.getAdditionalKanikoOptions().isEmpty())  {
+                    List<String> desiredOptions = Arrays.asList(dockerOutput.getAdditionalKanikoOptions().split("\\s+"));
```

Comment:
Is is possible for an option to contain whitespace? Even if none of them allow this today we can't rule it out in the future. Perhaps it would be better if the `addtionalOptions` were a list?

Reply:
I think with the `jvmOptions` we had the problem that list has not guaranteed ordering, right? But in general, I'm fine with list as well ... so I can change it.

Reply:
Ok, this should be now changed.